### PR TITLE
add migration warning

### PR DESCRIPTION
**Pull request by johanwelgemoed**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/operational-guides/update-guide/820-to-830.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ description: "Review which adjustments must be made to migrate from Camunda 8.2.
 
 The following sections explain which adjustments must be made to migrate from Camunda 8.2.x to 8.3.0 for each component.
 
+:::caution
+Updating Operate, Tasklist and Optimize from 8.2.x to 8.3.0 will potentially take longer than expected, depending on the data to be migrated.
+Additionally, we identified some bugs that could also prevent the migration from succeeding. These are being addressed and will be available in an upcoming 8.3.1 patch. We suggest not updating until the patch is released.
+After the patch, migration will succeed, but will still take long dependent on data to be migrated.
```
Comment (megglos) [inline docs/self-managed/operational-guides/update-guide/820-to-830.md:14]: I think we could skip that part, we indicate it takes a long time generally + hint on known issues to be resolved, thank you!
```suggestion
```

**Review by megglos — APPROVED** [review]
LGTM! @akeller & @christinaausley could you take a look as well and release it to 8.3.0 please?

**Review by pepopowitz — CHANGES_REQUESTED** [review]
Since we've cut version 8.3 already, this change will also need to be back-ported to the versioned_docs/version-8.3 source folder. 

@christinaausley I can take care of that if you'd like, but I'll wait for your review before copying it over.

**Comment by aabouzaid** [comment]
I've added it to the Helm upgrade page, and backported both changes to v8.3 dir :heavy_check_mark:

**Review by pepopowitz — DISMISSED** [review]

**Review by christinaausley — APPROVED** [review]

**Comment by akeller** [comment]
I am still about 500 GitHub notifications deep, but it looks like this needs to be merged and released today. Has this also been communicated to support and the community @johanwelgemoed?

**Pull request merged**